tests/test_TFUtil.py

```diff
@@ -469,6 +469,16 @@ def test_Data_copy_compatible_to_batch_feature_is_dynamic():
   assert_equal(t.get_size_dim_tag(0), dec.get_time_dim_tag())
 
 
+def test_Data_get_common_data():
```

Comment:
Please rename this function to something more specific. `test_Data_get_common_data_extra_static_spatial` or so.